tests/unit/amo/pages/TestUserProfile.js

```diff
@@ -830,23 +783,23 @@ describe(__filename, () => {
       reviews,
       count: DEFAULT_API_PAGE_SIZE + 2,
     });
-    const location = createFakeLocation({ query: { foo: 'bar' } });
 
-    const root = renderUserProfile({ location, params, store });
+    renderUserProfile({ params });
 
-    const paginator = shallow(root.find('.UserProfile-reviews').prop('footer'));
-    expect(paginator.instance()).toBeInstanceOf(Paginate);
-    expect(paginator).toHaveProp('count', DEFAULT_API_PAGE_SIZE + 2);
-    expect(paginator).toHaveProp('currentPage', '1');
-    expect(paginator).toHaveProp('pathname', `/user/${userId}/`);
-    expect(paginator).toHaveProp('queryParams', location.query);
+    const paginator = screen.getByClassName('UserProfile-reviews');
+    expect(paginator).toBeInTheDocument();
```

Comment:
I'm not sure we need this, if we are also asserting about something that is rendered by `paginator`.